node/common/core/package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "azure-iot-common",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
this will likely conflict with the new release - package versions should be bump at release time only.